core/domain/collection_services.py

```diff
@@ -1284,3 +1284,24 @@ def index_collections_given_ids(collection_ids):
     search_services.index_collection_summaries([
         collection_summary for collection_summary in collection_summaries
         if collection_summary is not None])
+
+
+def get_acquired_skills_of_user(user_id):
+    """Returns the acquired skills of the user identified by user_id.
+
+    Returns:
+        A list of skill ids acquired by the user.
+    """
+    completed_activities_model = (
+        user_models.CompletedActivitiesModel.get(
+            user_id, strict=False))
+    collection_ids = completed_activities_model.collection_ids
```

Comment:
I thought skills are supposed to only make sense within the context of a single collection?

Reply:
Yea, even I felt that way. I realised that there is possibility that there can be some skills which can be gained in one collection and can be used in other collection.
or
Do you think restricting it to context of a single collection is a better idea for now?

Reply:
Yes, I think skills are intended to be scoped to a collection (at least in the current design).

Reply:
Updated to scope to a collection.